src/libs/actions/Policy.js

```diff
@@ -57,6 +57,26 @@ function getSimplifiedEmployeeList(employeeList) {
     return employeeListEmails;
 }
 
+/**
+ * Used to update ALL of the policies at once. If a policy is present locally, but not in the policies object passed here it will be removed.
+ * @param {Object} policyCollection - object of policy key and partial policy object
+ */
+function updateAllPolicies(policyCollection) {
+    // Clear out locally cached policies that have been deleted (i.e. they exist locally but not in our new policy collection object)
+    _.each(allPolicies, (policy, key) => {
+        if (policyCollection[key]) {
+            return;
+        }
+
```

Comment:
NAB: Is this space necessary?

Reply:
No, I just prefer to pad if statements with new lines so things are less visually compressed.